native/src/main/java/io/ballerina/stdlib/http/api/service/signature/AllHeaderParams.java

```diff
@@ -158,7 +158,7 @@ private BMap<BString, Object> processHeaderRecord(HeaderParam headerParam, HttpH
                     return null;
                 } else {
                     String message = "no header value found for '" + key + "'";
```

Comment:
i guess we can make a constant string and use `String.format` in the used places

Reply:
Fixed via [77f7547](https://github.com/ballerina-platform/module-ballerina-http/pull/1846/commits/77f754789448ea6acaeb14b28a2d42d42e072a68)